.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "tests/testdata"]
 	path = tests/testdata
-	url = https://github.com/go-jet/jet-test-data
+	url = https://github.com/arjen-ag5/jet-test-data
```

Comment:
This change will need to be part of separate pull request for https://github.com/go-jet/jet-test-data repo.